ui/DesignSystem/Component/Timeline.svelte

```diff
@@ -0,0 +1,66 @@
+<script>
+  import { Text } from "../../DesignSystem/Primitive";
+  import ActionItem from "./Timeline/ActionItem.svelte";
+  import CommentItem from "./Timeline/CommentItem.svelte";
+  import CommentAction from "./Timeline/CommentAction.svelte";
+  export let startDate = null;
+  export let items = null;
+
+  // fake current user
+  const currentUser = {
+    handle: "julien",
+    avatar_url:
+      "https://avatars3.githubusercontent.com/u/2326909?s=24&u=1968a2daca982c6deaf89ec71c16d94333092fe3&v=4",
+  };
+</script>
+
+<style>
+  ul {
+    margin-top: 24px;
```

Comment:
Positioning and spacing information should come from the outside of a component to make it reusable in different places. So far we've been doing this via a `style` prop on the outer element:

```
<script>
  export let style = null;
</script>

<ul {style}>

</ul>
```

And then at the call site:
```
<Timeline style={margin: 24px 0 64px 0} />
```

Reply:
👍